app/models/host.rb

```diff
@@ -1568,6 +1524,33 @@ def scan_from_queue(taskid=nil)
     _log.info("Scanning #{log_target}...Complete - Timings: #{t.inspect}")
   end
 
+  def scan_refreshes(task)
+    # Firewall Rules and Advanced Settings go through EMS so we don't need Host credentials
+    scan_refresh("Firewall Rules", task) { refresh_firewall_rules }
+    scan_refresh("Advanced Settings", task) { refresh_advanced_settings }
+    unless ext_management_system
+      scan_refresh("IPMI information", task) { refresh_ipmi }
+    end
+  end
+
+  def scan_refreshes_with_connection(task, ssu)
```

Comment:
Love this PR, but not sure on the method name...connection is so generic that it doesn't explain what kind of connection, which to me makes the code unclear.  Maybe it's not important for others, but I think `scan_refreshes_over_ssh` or something like that might be clearer.

Also, should all these breakout methods be private?